chutney/packaging/local-api-unsecure/src/main/java/com/chutneytesting/SecurityConfiguration.java

```diff
@@ -13,16 +13,15 @@
 import java.util.Arrays;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.core.Ordered;
-import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
 import org.springframework.security.web.SecurityFilterChain;
 
 @Configuration
 @EnableWebSecurity
-@Order(Ordered.HIGHEST_PRECEDENCE)
+@EnableMethodSecurity
```

Comment:
- Are there tests to validate that method-level security is working correctly after this change?
- Have you checked whether @EnableMethodSecurity fully replaces the previous security mechanism?